### PR TITLE
added arch:ppc64le and permission fix for the same

### PR DESCRIPTION
**Pull request by ezeeyahoo**
Hi,

I request you to please review as the builds are passing, please let me know if there is more to be looked into.

Thanks

### CODE REVIEW

**Review by 4383 — APPROVED** [review]
Nice changes.

**Comment by TomSweeneyRedHat** [comment]
LGTM
@jwhonce PTAL.  @edsantiago or @cevich if you have a quick second for an eyeball...

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +10,10 @@ python:
   - '3.6'
   - '3.5'
 before_install:
+  - |
+    if [[ "${TRAVIS_CPU_ARCH}" == "ppc64le" ]]; then
+      sudo chown -Rvf $USER:$GROUP ~/.cache/pip/wheels;
```
Comment (edsantiago) [inline .travis.yml:15]: Could you add, somewhere in the commit message, exactly what problem this is trying to solve? As in, why is this directory not owned by the user, and why does that happen only on ppc64le?

Also, the mixture of `-v` and `-f` is weird enough to merit a comment (or removal). Maybe an explanation in the commit message, if it's hard to add comments in the yaml. Thanks!

Comment (4383) [reply .travis.yml:15]: I agree with @edsantiago, even an inline comment would be welcome.

Comment (cevich) [reply .travis.yml:15]: I also agree FWIW.

**Comment by cevich** [comment]
@ezeeyahoo out of curiosity, is this really running on ppc64 hardware in travis?  If so, cool!

**Comment by jwhonce** [comment]
@ezeeyahoo ping, would love to merge this once comments are addressed.  Thanks!

**Comment by ezeeyahoo** [comment]
@jwhonce Done 👍🏼

**Comment by ezeeyahoo** [comment]
I am trying without -f, I will remove if it goes fine on Travis

**Comment by ezeeyahoo** [comment]
Please let me know if this look messy and you want me to create a new PR, abandoning this one.

File: .travis.yml (hunk 1 above)
Comment (ezeeyahoo) [reply .travis.yml]: Done

**Comment by rhatdan** [comment]
Package is deprecated and being archived.

**Pull request closed without merging**